README.md

```diff
@@ -232,6 +232,12 @@ If you want the docker container to run in the background pass the `-d` flag rig
 You can adjust the maximum cache size by appending `--max_size=N`, where N is
 the maximum size in Gibibytes.
 
+### Kubernetes note 
+
+Don't name your deployment `bazel-remote`!
```

Comment:
nitpick: please add an empty line after this one

Reply:
ok